src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
as far as i remember few browse and screen reader combos respect aria-live tags, i think it might be almost completely pointless adding them in event though they are defined in the spec. which platforms, browsers and screen readers have you tested?